wp-content/themes/goonj-crm/functions.php

```diff
@@ -530,6 +555,28 @@ function goonj_redirect_after_individual_creation() {
 					);
 					break;
 				}
+				case 'institution-collection-camp':
+					if ( ! $source ) {
+						\Civi::log()->warning('Source is missing for material contribution flow', ['individualId' => $_GET['individualId']]);
+						return;
```

Comment:
why we are using the warning instead of `\Civi::log()->info` . Are we wan't to show anything on UI

Reply:
I think info is the right approach here. If there is no source, we can simply log it without triggering an error or any other action. I've updated the code.